<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortSettingDialog</class>
 <widget class="QDialog" name="SerialPortSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#SerialPortSettingDialog
{
border-radius：10px;
}

#btnClose{
border-image: url(:/icon/Dialog2/quxiao.png);
}
#label
{
border-image: url(:/icon/Dialog2/shezhi.png);
}
#widget{
border-image: url(:/icon/Dialog2/chuankoukuang.png);
padding-left:2px;
padding-right:4px;
padding-top:6px;
padding-bottom:6px;
}
#label_2,#label_3,#label_4,#label_5,#label_6{
font: 12pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
}


QComboBox  {
border:none;
font: 12pt &quot;微软雅黑&quot;;
background-color: rgb(0, 0, 0);
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: right;  
    border:none;
    width:30%;
    height:30%
}
QComboBox::down-arrow {
	image:url(:/icon/Dialog2/zhankai.png)
}
#widget_2{
border-image: url(:/icon/Dialog2/fasong.png);
}
#widget_2:hover{
border-image: url(:/new/prefix1/fasong2x_1.png);
}


#widget_3{
border-image: url(:/icon/Dialog2/dingshifasong.png)
}
#widget_3:hover{border-image: url(:/new/prefix1/dingshifasong2x_1.png);}

#widget_4{
border-image: url(:/icon/Dialog2/dakaiguanbi.png);
}
#widget_4:hover{border-image: url(:/new/prefix1/dakaiguanbi2x_1.png);}

#widget_5{
border-image: url(:/icon/Dialog2/tingzhi.png);
}
#widget_5:hover{border-image: url(:/new/prefix1/tingzhi_1.png);}


#btnTimerSend{
border-radius:41px;
}
#btnSend{
border-radius:41px;
}
#btnOpenSerial{
border-radius:41px;
}
#btnStopSend{
border-radius:41px;
} </string>
  </property>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>627</x>
     <y>25</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>83</y>
     <width>320</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>145</y>
     <width>340</width>
     <height>434</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>57</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>串口号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>347</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>129</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>检验位</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>202</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>停止位</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>274</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>数据位</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>350</y>
      <width>140</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>270</y>
      <width>140</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>198</y>
      <width>140</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>125</y>
      <width>140</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>53</y>
      <width>140</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label_6</zorder>
   <zorder>label_5</zorder>
   <zorder>label_4</zorder>
   <zorder>label_3</zorder>
   <zorder>comboBox_4</zorder>
   <zorder>comboBox_3</zorder>
   <zorder>comboBox_2</zorder>
   <zorder>comboBox</zorder>
   <zorder>comboBox_5</zorder>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>145</y>
     <width>281</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>243</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>251</y>
     <width>281</width>
     <height>125</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btnTimerSend">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>243</width>
      <height>82</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>357</y>
     <width>281</width>
     <height>125</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btnOpenSerial">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>243</width>
      <height>82</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>463</y>
     <width>281</width>
     <height>125</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btnStopSend">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>243</width>
      <height>82</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
